components/automate-builder-api-proxy/habitat/config/mime.types

```diff
@@ -0,0 +1,89 @@
+
+types {
```

Comment:
I think that this file also lives in the nginx package that we ship, so unless we are doing something special, we could do something like:

```
components/automate-load-balancer/habitat/config/nginx.conf
12:  include        {{ pkgPathFor "core/nginx" }}/config/mime.types;
```

and exclude this file.